src/discovery.rs

```diff
@@ -412,7 +409,7 @@ impl Context for OidcDiscovery {
                             open_id_config: Arc::new(OpenIdConfig {
                                 auth_endpoint: auth_endpoint.clone(),
                                 token_endpoint: token_endpoint.clone(),
-                                issuer: issuer.clone(),
+                                issuer: issuer.to_string(),
```

Comment:
Why `to_string`? It's already a string. I feel like `clone` expresses better what we want here.